### PR TITLE
Initial working simulator

### PR DESCRIPTION
**Pull request by roni-kreinin**
Depends on https://github.com/clearpathrobotics/clearpath_common/pull/12

Basic teleop works with A200.

### CODE REVIEW

File: launch/clearpath_simulator.launch.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+
+ARGUMENTS = [
+    DeclareLaunchArgument('namespace', default_value='',
+                          description='Robot namespace'),
+    DeclareLaunchArgument('rviz', default_value='false',
+                          choices=['true', 'false'], description='Start rviz.'),
+    DeclareLaunchArgument('world', default_value='warehouse',
+                          description='Gazebo World'),
+    DeclareLaunchArgument('model', default_value='standard',
```
Comment (tonybaltovski) [inline launch/clearpath_simulator.launch.py:33]: I think these need to be updated.

File: launch/gz_sim.launch.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+from launch_ros.actions import Node
+
+
+ARGUMENTS = [
+    DeclareLaunchArgument('use_sim_time', default_value='true',
+                          choices=['true', 'false'],
+                          description='use_sim_time'),
+    DeclareLaunchArgument('world', default_value='warehouse',
+                          description='Gazebo World'),
+    DeclareLaunchArgument('model', default_value='lite',
```
Comment (tonybaltovski) [inline launch/gz_sim.launch.py:37]: Update for Clearpath Platforms.

File: launch/robot_spawn.launch.py [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+
+ARGUMENTS = [
+    DeclareLaunchArgument('rviz', default_value='false',
+                          choices=['true', 'false'],
+                          description='Start rviz.'),
+    DeclareLaunchArgument('use_sim_time', default_value='true',
+                          choices=['true', 'false'],
+                          description='use_sim_time'),
+    DeclareLaunchArgument('model', default_value='standard',
```
Comment (tonybaltovski) [inline launch/robot_spawn.launch.py:39]: Update for Clearpath Platforms.

File: launch/robot_spawn.launch.py [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+                          choices=['standard', 'lite'],
+                          description='Turtlebot4 Model'),
+    DeclareLaunchArgument('namespace', default_value='',
+                          description='Robot namespace'),
+    DeclareLaunchArgument('localization', default_value='false',
```
Comment (tonybaltovski) [inline launch/robot_spawn.launch.py:44]: Should localization, slam and nav2 be moved out to its own launch files?

**Review by tonybaltovski — CHANGES_REQUESTED** [review]

File: launch/gz_sim.launch.py (hunk 2 above)
Comment (tonybaltovski) [reply launch/gz_sim.launch.py:37]: @roni-kreinin this change?

Comment (roni-kreinin) [reply launch/gz_sim.launch.py]: Fixed.

**Review by tonybaltovski — APPROVED** [review]

**Pull request merged**